langchain4j-vertex-ai-gemini/src/main/java/dev/langchain4j/model/vertexai/VertexAiGeminiChatModel.java

```diff
@@ -59,7 +73,13 @@ public VertexAiGeminiChatModel(String project,
                                    Integer maxOutputTokens,
                                    Integer topK,
                                    Float topP,
-                                   Integer maxRetries) {
+                                   Integer maxRetries,
+                                   String responseMimeType,
```

Comment:
please add Gemini to the list here: https://docs.langchain4j.dev/tutorials/ai-services#json-mode

Reply:
Actually, yes, since that's part of the sources, I should update it right away.